Coupled-Cluster/Spin_Orbitals/CCSD/CCSD.py

```diff
@@ -89,7 +89,7 @@
 print('..finished transformation in %.3f seconds.\n' % (time.time() - t))
 
 # DPD approach to CCSD equations
-# See: http://github.com/CrawfordGroup/ProgrammingProjects
+# See: http://sirius.chem.vt.edu/wiki/doku.php?id=crawdad:programming
```

Comment:
Hmm, I remember someone switched this over to the GitHub repo. Is this a conscious deprecation?

Reply:
Nope. FIxed.